src/Microsoft.Azure.SignalR/ServerConnections/ServiceConnection.cs

```diff
@@ -140,7 +142,7 @@ protected override Task OnClientConnectedAsync(OpenConnectionMessage message)
         protected override Task OnClientDisconnectedAsync(CloseConnectionMessage closeConnectionMessage)
         {
             var connectionId = closeConnectionMessage.ConnectionId;
-            if (_enableConnectionMigration && _clientConnectionManager.ClientConnections.TryGetValue(connectionId, out var context))
+            if (EnableMigration && _clientConnectionManager.ClientConnections.TryGetValue(connectionId, out var context))

```

Comment:
>EnableMigration [](start = 16, length = 15)

looks EnableMigration only used here, why not directly check `use _migrationLevel != ServerConnectionMigrationLevel.Off` ?

Reply:
Much easier to understand if I introduced this property?

Reply:
temp variable should be enough here?

---
In reply to: [396882589](https://github.com/Azure/azure-signalr/pull/857#discussion_r396882589) [](ancestors = 396882589)